android/src/main/java/co/nimblehq/ic/kmm/suv/android/ui/navigation/Navigation.kt

```diff
@@ -0,0 +1,23 @@
+package co.nimblehq.ic.kmm.suv.android.ui.navigation
+
+import androidx.compose.runtime.Composable
+import androidx.navigation.compose.NavHost
+import androidx.navigation.compose.composable
+import androidx.navigation.compose.rememberNavController
+import co.nimblehq.ic.kmm.suv.android.ui.screens.main.MainScreen
+import co.nimblehq.ic.kmm.suv.android.ui.screens.splash.SplashScreen
+
+@Composable
+fun Navigation() {
+    val navController = rememberNavController()
+    NavHost(navController = navController, startDestination = Screens.SPLASH) {
```

Comment:
I think it's better to define the `navController` and start `startDestination` if you want to make it scalable, we can inject different NavController for usage.

Reply:
Improved by using `NavGraphBuilder`

For this project, I would prefer to use the `NavGraphBuilder` approach. I will improve this later when the android-templates define the template for compose navigation 🙌

<img width="889" alt="Screen Shot 2022-09-21 at 12 31 37" src="https://user-images.githubusercontent.com/19943832/191421948-b971e3f5-5fbd-4dc1-9513-5abcb9c780c7.png">

Fixed in https://github.com/suho/kmm-ic/pull/79/commits/30bef640c4329f8860ab9d30b9e6134791383f58